data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+		"Fragmentation Grenades" 20
+		"Pulse Rifle" 12
+		"Outfits Expansion" 2
+		`"Bufaer" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Hyperdrive"
+	cargo
+		outfits
+			"Building Materials" 190
+		commodities
+			"Equipment" 100
+			"Electronics" 100
+			"Food" 100
+
+mission "<uhai> 7: Stolen Cargo Notification"
+	minor
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "event: unfettered: shipment lost"
+		not "<uhai> 7: Stolen Cargo: active"
+		not "<uhai> 7: Stolen Cargo: done"
+		random < 60
+	on offer
+		conversation
+			`Upon landing you take a look at the job board, almost by habit, and to your surprise you notice a very peculiar job, left by none other than Sookootow, titled "looking for stolen cargo".`
+			`	It does not look like an urgent job, so it should still be there the next time you land on an Unfettered world, but you should probably check it out when you have the time.`
+				decline
+
+# SPOILER: Cerberus or Midas bring cargo to the Unfettered. Cerberus was on the way, but Midas hired Scar's legion to sneakily take it and bring it to them.
+# only that Cerberus were on their tail, and just took down the ship and it's cargo when you arrive. Time to steal it back back back
+mission "<uhai> 7: Stolen Cargo"
+	job
+	name `Looking for stolen cargo (Sookootow)`
+	description `Apt captain required to find the pirate vessel that has stolen the materials needed for the development of the urban areas. Bring 142 tons back to <destination>, the ship may be found in north-east or north-west pirate space.`
+	clearance `As soon as you say that you are here on behalf of the Unfettered, you are cleared to land.`
+	source
+		attributes "unfettered"
+	to offer
+		has "event: unfettered: shipment lost"
+	substitutions
+		<abducted> "only wishing to scare us with the possibility of abducting"
+		<abducted> "although to be completely frank I have no clue how they managed to abduct"
+			has "<uhai> 6 A: Loyal Escort: failed"
+	on accept
+		conversation
+			`	You go to see Sookootow in order to discuss this job in greater details, and easily find him by asking around, everyone seems to know him here. He was having a quite animated conversation with an elder, but he puts a stop to it when he notices you.`
+			`	"<last>. I am still saddened by what happened to that governor you had to escort... She used to say that all Hai are together, by heart. I will continue to honor her memories."`
+				to display
+					has "<uhai> 6 A: Loyal Escort: failed"
+			`	"<last>! I was hoping to find you here."`
+				to display
+					has "<uhai> 6 A: Loyal Escort: done"
+			`	"I just happen to be in needs of your services. Remember the mission you did earlier, escorting Hai ships? Well that shipment was not the only one that got attacked by pirates, in fact I believe their attack on it may have been nothing more than a distraction; they knew full well they could not take that cargo in the middle of Hai space, <abducted> the governor."`
```

Comment:
```suggestion
			`	"I just happen to be in need of your services. Remember the mission you did earlier, escorting Hai ships? Well that shipment was not the only one that got attacked by pirates, in fact I believe their attack on it may have been nothing more than a distraction; they knew full well they could not take that cargo in the middle of Hai space, <abducted> the governor."`
```